src/TextInput/TextInput.svelte

```diff
@@ -94,6 +124,14 @@
     dispatch("change", parse(e.target.value));
```

Comment:
I think these `onChange` and `parse` methods should be removed. We cast
`number`-type inputs, but the React library doesn't. Maybe this can be handled
in userland, especially since `<input type="number" />` when submitted in a
`<form>` is read server-side as a string.

Demo: https://www.sveltelab.dev/csng8xsyb7bhy56

cc @metonym

Reply:
I'd agree that it would align better with the user expected type especially since there is a separate `NumberInput` component.

Lemme know if you want me to press ahead with that change.